test/integration/tlspolicy_controller_test.go

```diff
@@ -326,18 +362,121 @@ var _ = Describe("TLSPolicy", Ordered, func() {
 						return fmt.Errorf("expected CertificateList to be 2")
 					}
 					return nil
-				}, time.Second*10, time.Second).Should(BeNil())
+				}, time.Second*30, time.Second).Should(BeNil())
 
 				cert1 := &certmanv1.Certificate{}
 				err := k8sClient.Get(ctx, client.ObjectKey{Name: "test-tls-secret", Namespace: testNamespace}, cert1)
 				Expect(err).ToNot(HaveOccurred())
 
+				Expect(cert1.Spec.DNSNames).To(BeEquivalentTo([]string{"test1.example.com", "test2.example.com"}))
```

Comment:
Will `BeEquivalentTo` take into account the ordering of the elements? `ConsistOf` is another matcher that checks elements exist and only those elements in an array regardless of order.
```
ConsistOf("test1.example.com", "test2.example.com")
```

Reply:
I had assumed it would based on the name, but I'm not sure, I'll update it to `ConsistOf`